src/main/java/com/fibermc/essentialcommands/commands/utility/SuicideCommand.java

```diff
@@ -21,7 +21,7 @@ public int run(CommandContext<ServerCommandSource> context) throws CommandSyntax
             return 0;
         }
 
-        player.kill();
+        player.kill(player.server.getWorld(player.getWorld().getRegistryKey()));
```

Comment:
lmao

Reply:
yeah so apparently players can now not be in a world

Reply:
Mojang, what? (`Entity.java`)

![image](https://github.com/user-attachments/assets/50215b87-a0c7-4c71-b832-78a7a5c7b516)

For now I'll just recommend doing `player.getServerWorld()` since we know this is a `ServerPlayerEntity` already.

Reply:
> players can now not be in a world

I somehow read the as the opposite of what you wrote. If that's actually a thing, maybe steal what `/kill` does - `source.getContext().getWorld()`